### PR TITLE
199 fix create issue modal window

### PR DESCRIPTION
**Pull request by Vadis00**
[199-fix-create-issue-modal-window](https://trello.com/c/jiN3BMgF/199-fix-create-issue-modal-window)

### CODE REVIEW

**Review by andpii — DISMISSED** [review]

File: frontend/src/shared/components/tasque-task-creation/task-creation.component.html [hunk 1]
```diff
@@ -3,57 +3,41 @@
...
-              [options]="projects"
-              formControlName="projectControl"
-              [errorMessage]="projectErrorMessage"
-              [invalid]="!projectControl.valid"
-            >
-            </tasque-dropdown>
+            <div class="label-wrapper">
+              <label>Project</label>
+            </div>
+            <tasque-dropdown [options]="projects" formControlName="projectControl" [errorMessage]="projectErrorMessage" [invalid]="!projectControl.valid"> </tasque-dropdown>
```
Comment (ismv) [inline frontend/src/shared/components/tasque-task-creation/task-creation.component.html]: formatting

File: frontend/src/shared/components/tasque-task-creation/task-creation.component.html [hunk 2]
```diff
@@ -3,57 +3,41 @@
...
-              [options]="issueTypes"
-              [formControl]="issueTypeControl"
-              [errorMessage]="issueTypeErrorMessage"
-              [invalid]="!issueTypeControl.valid"
-            >
-            </tasque-dropdown>
+            <div class="label-wrapper">
+              <label>Issue type</label>
+            </div>
+            <tasque-dropdown [options]="issueTypes" [formControl]="issueTypeControl" [errorMessage]="issueTypeErrorMessage" [invalid]="!issueTypeControl.valid"> </tasque-dropdown>
```
Comment (ismv) [inline frontend/src/shared/components/tasque-task-creation/task-creation.component.html]: same

File: frontend/src/shared/components/tasque-task-creation/task-creation.component.html [hunk 3]
```diff
@@ -3,57 +3,41 @@
...
-              [formControl]="summaryControl"
-              [errorMessage]="summaryErrorMessage"
-              [invalid]="!summaryControl.valid"
-              ngDefaultControl
-            >
-            </tasque-input>
+            <div class="label-wrapper">
+              <label>Summary</label>
+            </div>
+            <tasque-input class="input" [formControl]="summaryControl" [errorMessage]="summaryErrorMessage" [invalid]="!summaryControl.valid"> </tasque-input>
```
Comment (ismv) [inline frontend/src/shared/components/tasque-task-creation/task-creation.component.html]: same

File: frontend/src/shared/components/tasque-task-creation/task-creation.component.ts [hunk 4]
```diff
@@ -68,7 +72,7 @@ export class TaskCreationComponent implements OnInit, OnDestroy {
   get descriptionErrorMessage(): string {
     const ctrl = this.descriptionControl;
 
-    if (ctrl.errors?.['maxlength']) {
+    if (ctrl.errors?.['maxlength'] && (ctrl.dirty || ctrl.touched)) {
       return 'Description must be at less  80 characters';
```
Comment (Its-OP) [inline frontend/src/shared/components/tasque-task-creation/task-creation.component.ts:76]: At least

File: frontend/src/shared/components/tasque-task-creation/task-creation.component.ts [hunk 5]
```diff
@@ -78,7 +82,7 @@ export class TaskCreationComponent implements OnInit, OnDestroy {
   get editorErrorMessage(): string {
     const ctrl = this.descriptionControl;
 
-    if (ctrl.errors?.['maxlength']) {
+    if (ctrl.errors?.['maxlength'] && (ctrl.dirty || ctrl.touched)) {
       return 'Description must be at less 5000 characters';
```
Comment (Its-OP) [inline frontend/src/shared/components/tasque-task-creation/task-creation.component.ts:86]: must not exceed

**Review by Its-OP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**